docs/contributing.rst

```diff
@@ -62,6 +62,10 @@ Finally, please make sure you respect the `coding style
 for this project. Also, even though we do CI testing, please test your code and
 ensure that it builds locally before submitting a pull request.
 
+Please note that this project is released with a `Contributor Code of Conduct
+<https://raw.githubusercontent.com/ethereum/solidity/develop/CONDUCT.md>`_.
```

Comment:
We could also use the `include` statement here to pull in the markdown, though it wouldn't be parsed properly I guess.

Reply:
Actually, it looks like all the items on the checklist are also addressed by this section. Also, I had a look at the current state of sphinx and rendering markdown is not supported yet. There is a workaround but that looks rather complicated ;)